server/cli/src/cli.rs

```diff
@@ -209,11 +212,18 @@ enum Action {
         #[clap(short, long, action = ArgAction::SetTrue, conflicts_with="enable")]
         disable: bool,
     },
-    /// Generate TypeScript Types for backend plugins and format with Prettier
-    GenerateTypeScriptTypes,
-    /// Run load test
-    #[cfg(feature = "integration_test")]
-    LoadTest(LoadTest),
+    GeneratePluginTypescriptTypes {
```

Comment:
This rename makes good sense, however could it break any existing workflows or docs that need updating?

Reply:
I think it would be ok, i couldn't find any docs that mention this, it's not mentioned here anyways: https://github.com/andreievg/open-msupply-actions-test/blob/2bebddeaa46ed7baac1c6556435507bd018941d3/server/README.md#L332-L351

I have started using this now in the plugin: https://github.com/msupply-foundation/sao-tome-plugins/blob/43236f3a9401e4592aedfadb6e2649942abec89d/test/src/test_lib/build_plugin_bundle.rs#L52-L63 so hope it's ok to keep.

I am glad you made a comment here since the line above removes load test as a variant, and compiler doesn't complain because it's behind integration test, reinstated here
2bebddeaa46ed7baac1c6556435507bd018941d3